cli/popper/commands/cmd_stages.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+
+import click
+import popper.utils as pu
+
+from popper.cli import pass_context
+
+@click.command(
+    'stages', short_help='See and manipulate stages of a pipeline.'
+)
+@click.argument('pipeline', required=True)
+@click.option(
+    '--set',
+    help='Provide comma separated list of stages to set the stages of a '
+         'pipeline. Without the --replace flag, the stages are added alongwith '
+         'the existing stages, otherwise they are replaced.',
+    required=False,
+)
+@click.option(
+    '--replace',
+    is_flag=True,
+    help='Use alongwith the --set option to overwrite the curent stages.',
+    required=False,
+)
```

Comment:
I'd vote for removing this flag. I think the most frequent case is to overwrite the stages